pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -323,11 +437,7 @@ func (r *ReconcileMachineHealthCheck) reconcileStatus(baseToPatch client.Patch,
 
 // healthCheckTargets health checks a slice of targets
 // and gives a data to measure the average health
-func (r *ReconcileMachineHealthCheck) healthCheckTargets(targets []target, timeoutForMachineToHaveNode time.Duration) (int, []target, []time.Duration, []error) {
-	var nextCheckTimes []time.Duration
-	var errList []error
-	var needRemediationTargets []target
-	var currentHealthy int
+func (r *ReconcileMachineHealthCheck) healthCheckTargets(targets []target, timeoutForMachineToHaveNode time.Duration) (currentHealthy []target, needRemediationTargets []target, nextCheckTimes []time.Duration, errList []error) {
```

Comment:
Please don't use named returns, it's confusing and error prone. Be explicit in each return statement please